homeassistant/components/nuki/__init__.py

```diff
@@ -101,42 +70,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except RequestException as err:
         raise exceptions.ConfigEntryNotReady from err
 
-    async def async_update_data() -> None:
-        """Fetch data from Nuki bridge."""
-        try:
-            # Note: asyncio.TimeoutError and aiohttp.ClientError are already
-            # handled by the data update coordinator.
-            async with async_timeout.timeout(10):
-                events = await hass.async_add_executor_job(
-                    _update_devices, locks + openers
-                )
-        except InvalidCredentialsException as err:
-            raise UpdateFailed(f"Invalid credentials for Bridge: {err}") from err
-        except RequestException as err:
-            raise UpdateFailed(f"Error communicating with Bridge: {err}") from err
-
-        ent_reg = er.async_get(hass)
-        for event, device_ids in events.items():
-            for device_id in device_ids:
-                entity_id = ent_reg.async_get_entity_id(
-                    Platform.LOCK, DOMAIN, device_id
-                )
-                event_data = {
-                    "entity_id": entity_id,
-                    "type": event,
-                }
-                hass.bus.async_fire("nuki_event", event_data)
-
-    coordinator = DataUpdateCoordinator(
-        hass,
-        _LOGGER,
-        # Name of the data. For logging purposes.
-        name="nuki devices",
-        update_method=async_update_data,
-        # Polling interval. Will only be polled if there are subscribers.
-        update_interval=UPDATE_INTERVAL,
+    # Device registration for the bridge
```

Comment:
A bridge (the 3.0 which is in sell now long time) can have more than 1 lock device. Does that still work?

Reply:
Mine has a lock + opener connected, that works. I guess the limitation here is that the integration can only connect to a single bridge, but since the integration can be created multiple times I don't see this as an issue.

Reply:
I've just tried it with a mock API to have one opener and two locks, works flawlessly for me
![image](https://user-images.githubusercontent.com/9799276/214687573-f338c79d-81f8-4496-a436-140380d459d3.png)